README.md

```diff
@@ -92,3 +92,11 @@ git checkout my-pr-branch
 git rebase develop
 git push origin my-pr-branch --force[-with-lease]
 ```
+
+### Running Danger Locally
+
+You can run danger locally to see what it would output by running the following in the root of the repository.
+
+```bash
+swift run danger-swift pr [some-pr-url]
```

Comment:
🔥 would it be possible to run it locally without a PR URL

Reply:
@cyndichin yea. `swift run danger-swift local` should work, but haven't used it!

Reply:
Actually i lied, it fails because it tries to do git things.

Reply:
ah bummer :(